inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/manager/enums/WorkflowType.java

```diff
@@ -15,24 +15,31 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package de.tudarmstadt.ukp.inception.workload.dynamic.manager;
+package de.tudarmstadt.ukp.inception.workload.dynamic.manager.enums;
 
-import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.stereotype.Component;
+import de.tudarmstadt.ukp.clarin.webanno.support.PersistentEnum;
 
-@Component
-@ConfigurationProperties("defaultannotations")
-public class DefaultAnnotationsPropertiesImpl implements DefaultAnnotationsProperties {
+public enum WorkflowType implements PersistentEnum
+{
+    RANDOMIZED_WORKFLOW_TYPE("RANDOMIZED_WORKFLOW_TYPE"),
+    DEFAULT_WORKFLOW_TYPE("DEFAULT_WORKFLOW_TYPE");
```

Comment:
Is it possible to move enum somewhere else, e.g. the central/default workflow place? The dynamic workflow module should not define the default one. Also, I think @reckart  does not like enums but would prefer string constants because new modules can be added without needing to add another enum entry.